Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -222,7 +232,6 @@ public void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.EventFunction<
 
                 // If there is a focused object in the hierarchy of the modal handler, start the event
                 // bubble there
-                GameObject focusedObject = GazeManager.Instance.HitObject;
```

Comment:
Doesn't seem like the value of the focused object wouldn't matter if it was assigned before checking the stack count, so it should be okay.